packages/react-components/src/radio/src/Radio.css

```diff
@@ -0,0 +1,145 @@
+:root {
+    --outline: 0 0 0 2px var(--primary-100);
```

Comment:
I believe it should be scoped to our components, we want to avoid as much as possible leaking in the consumer apps.

You still have the "index.css" file in which we can define global css properties for our components.

I believe you might want to use a scoped prop name like `--o-ui-outline` though to avoid clashing with consumer vars if they add a custom class to any of the components.

Reply:
By sitewide I meant in index.css.